README.md

```
@@ -1,212 +1,110 @@
-# Clone the repository
-git clone https://github.com/felores/airtable-mcp.git
-cd airtable-mcp
+3. **Start de server**
+   ```bash
+   npm run dev
+   ```
 
-# Install dependencies
-npm install
+4. **Open je browser**
+   ```
```

markdownlint-cli2 (0.20.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)